app/services/fetch_coronavirus_statistics_service.rb

```diff
@@ -105,15 +108,53 @@ def latest_vaccinations(data)
 
   def latest_admissions(data)
     latest_admissions = data.find { |d| d["hospitalAdmissions"] }
-
     return {} unless latest_admissions
 
+    daily_admissions = daily_admissions(latest_admissions)
+    weekly_admissions = weekly_admissions(latest_admissions, data)
+
+    daily_admissions.merge!(weekly_admissions)
+  end
+
+  def daily_admissions(latest_admissions)
     {
       hospital_admissions: latest_admissions["hospitalAdmissions"],
       hospital_admissions_date: Date.parse(latest_admissions["date"]),
     }
   end
 
+  def weekly_admissions(latest_admissions, data)
+    weeks_with_admissions = []
```

Comment:
not a blocking comment but I'm not sure `weeks_with_admissions` really sums up what this is doing, it's more like `daily_admissions_list` or `daily_admissions`

Reply:
which makes lines 132 and 133 read a little more clearly, but again, super minor